manifests/plugin/dynflow.pp

```diff
@@ -61,7 +61,7 @@
     }
 
     foreman_proxy::plugin { 'dynflow_core':
-      package => "${scl_prefix}${::foreman_proxy::plugin_prefix}dynflow_core",
+      package => "${scl_prefix}rubygem-smart_proxy_dynflow_core",
```

Comment:
Can't this just be:smart_proxy_dynflow_core",
```suggestion
      package => "${foreman_proxy::plugin_prefix}dynflow_core",
```

Reply:
I suppose it could be now. I was trying to leave it robust against the incoming changes rather than it only work in the SCL based case. The former would allow us to merge this code, and support SCL for testing. I personally don't see what it hurts to hard code this part here; especially in light of how special the `_core` packages are and my attempts to see if we can design them better.

Reply:
My reasoning would mostly be that this breaks on Debian. That means you're going to replicate the `plugin_prefix` anyway. However, currently we don't deploy it on Debian and maybe we should use the cleanup of the `_core` situation to align the two platforms.

Reply:
You are right... I am already breaking compatibility with the change to `plugin_prefix` naming. This is updated now.